frontend/src/models/models.html

```diff
@@ -66,7 +66,7 @@ <h1>Models</h1>
       <template v-slot:body>
         <div class="modal-exit" @click="shouldShowFieldModal = false; selectedPaths = [...filteredPaths];">&times;</div>
         <div v-for="(path, index) in schemaPaths" :key="index" style="margin-bottom: 0.5em">
-          <input type="checkbox" :id="'path.path'+index" @change="addOrRemove(path)" :value="path.path" :checked="isSelected(path.path)"/>
+          <input type="checkbox" :id="'path.path'+index" @change="addOrRemove(path)" :value="path.path" :checked="isSelected(path.path)" :render="rerenderCheckboxes"/>
```

Comment:
This is a sloppy hack. You're introducing a separate variable that is practically identical to `shouldShowFieldModal`, and all this variable does is hide the underlying bug that the logic in `filterDocuments()` that converts `selectedPaths` to `filteredPaths` assumes that `selectedPaths` is an object, not an array. Take a look at https://github.com/mongoosejs/studio/pull/21/commits/0fc762d7cfafeea3a6d640d3d096aa89efb7241d#diff-f517035342619d6bdd368c77bc83b14c5293d02c1e262cdc302126e9bafe5128 for the fix